pkg/cli/cmd/directory/object.go

```diff
@@ -0,0 +1,253 @@
+package directory
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/aserto-dev/clui"
+	"github.com/aserto-dev/go-directory/aserto/directory/common/v3"
+	"github.com/aserto-dev/go-directory/aserto/directory/reader/v3"
+	"github.com/aserto-dev/go-directory/aserto/directory/writer/v3"
+	"github.com/aserto-dev/topaz/pkg/cli/cc"
+	"github.com/aserto-dev/topaz/pkg/cli/clients"
+	"github.com/aserto-dev/topaz/pkg/cli/jsonx"
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/structpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type GetObjectCmd struct {
+	Request  string `arg:""  type:"existingfile" name:"request" optional:"" help:"file path to get object request or '-' to read from stdin"`
```

Comment:
Feels odd that users would have to do something like:
```
echo '{"object_type": "user", "object_id": "ronen@aserto.com"}' | topaz directory get-object -`
```
In order to get an object from the directory.
I would expect something like:
```
topaz directory get-object user ronen@aserto.com
```
or
```
topaz directory get-object --type user --id ronen@aserto.com
```

Reply:
Modified to allow using directory commands also like, while keeping current functionality:
`topaz directory get-object '{"object_type":"user","object_id":"ronen@aserto.com"}'`